flow.py

```diff
@@ -50,7 +50,7 @@ class FlowBase(ABC):  # pylint: disable=too-many-instance-attributes
     _flow_mod_class = None
     _match_class = None
 
-    def __init__(self, switch, table_id=0xff, match=None, priority=0,
+    def __init__(self, switch, table_id=0xff, match=None, priority=0x8000,
```

Comment:
As there is no default value defined in the Openflow spec AFAIK, this is rather an arbitrary value, right? It is important to keep this in mind because:
1. NApps that install flows need to have the priority configured individually in case of explicit precedence. Setting the default to zero added implicit lowest priority to flows without explicit configuration.
2. Kytos NApps that rely in _table-miss flows_, as [the L2 switch](https://github.com/kytos/of_l2ls) for instance (but there may be others) will need the zero value explicitly defined.

Reply:
It is arbitrary, as using zero is also arbitrary, but `0x8000` is the default value used in all implementations I've seen, like _Ryu_, _NOX_ and also in _OVS_.
IMHO, the desired behavior is exactly what you addressed in 1. Flows installed with the default should never have the maximum nor the minimum possible priority.
We really need to take care with the NApps that rely on the default value of zero in the priority.